data/announcements/scheduled.yaml

```diff
@@ -118,7 +118,7 @@ announcements:
     May 16 - 20, 2022.
 
 - name: Kubecon 2022 NA
-  startTime: 2022-10-17T00:00:00 # Added in https://github.com/kubernetes/website/pull/36107 
+  startTime: 2022-09-23T00:00:00 # Added in https://github.com/kubernetes/website/pull/36107 
```

Comment:
```suggestion
  startTime: 2022-09-23T00:00:00 # Added in https://github.com/kubernetes/website/pull/36107
                                 # Date amended in https://github.com/kubernetes/website/pull/36927
```
?

Reply:
oops, thx

Reply:
There are a few of these in this file -- we may want to go and do a cleanup pass at some point.